.github/workflows/ci-ristretto-lint.yml

```diff
@@ -0,0 +1,27 @@
+name: ci-ristretto-lint
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "0 * * * *"
+jobs:
+  go-lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golang-lint
+        env:
+          # prevent OOM
+          GOGC: 10
+        uses: golangci/golangci-lint-action@v2
```

Comment:
to control what linters run, you will need to add something like this https://github.com/dgraph-io/dgraph/blob/main/.golangci.yml

Reply:
Done (only running two lint tests now as in dgraph but we can open it up later)